### PR TITLE
[Feature](mlu-ops): Add trademark for MLU-OPS repository.

### PR DESCRIPTION
**Pull request by Unireverse**
Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation

Add trademark for MLU-OPS.

## 2. Modification

	modified:   .github/ISSUE_TEMPLATE/feature_request.md
	modified:   .github/pull_request_template.md
	modified:   CONTRIBUTION.md
	modified:   README.md
	modified:   docs/GTest-User-Guide-zh.md
	modified:   docs/Gencase-User-Guide-zh.md
	modified:   docs/MLU-OPS-Accuracy-Acceptance-Standard.md
	modified:   docs/MLU-OPS-Compile-Develop-And-Test.md
	modified:   docs/MLU-OPS-How-To-Use-CNNL-API.md
	modified:   docs/MLU-OPS-OpList.md
	modified:   docs/MLU-OPS-Operator-Design-Doc-Template.md
	modified:   docs/MLU-OPS-Operator-Development-Process.md
	modified:   docs/MLU-OPS-Performance-Acceptance-Standard.md
	modified:   docs/MLU-OPS-Test-Report-Template.md
	modified:   docs/Pull-Request.md
	modified:   docs/user_guide/11_env_var/index.rst
	modified:   docs/MLU-OPS-LOGO.png

## 3. Test Report

No test.

### CODE REVIEW

**Review by PetrelYy — APPROVED** [review]

**Review by mahxn0 — APPROVED** [review]

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
     - 算子伪代码开发
     - 算子主体代码开发
 
-3. 完成测试并撰写测试报告，参考[MLU-OPS性能验收标准](docs/MLU-OPS-Performance-Acceptance-Standard.md)、[MLU-OPS测试报告模板](docs/MLU-OPS-Test-Report-Template.md)、[MLU-OPS-Accuracy-Acceptance-Standard](docs/MLU-OPS-Accuracy-Acceptance-Standard.md)，主要包括：
+3. 完成测试并撰写测试报告，参考[MLU-OPS性能验收标准](docs/MLU-OPS™-Performance-Acceptance-Standard.md)、[MLU-OPS™测试报告模板](docs/MLU-OPS-Test-Report-Template.md)、[MLU-OPS™-Accuracy-Acceptance-Standard](docs/MLU-OPS-Accuracy-Acceptance-Standard.md)，主要包括：
```
Comment (mahxn0) [inline CONTRIBUTION.md:24]: 这里改错了 文件名不能加

**Review by DanieeelLiu — APPROVED** [review]

**Comment by chqy99** [comment]
精度文档最大单点误差和综合单点误差需要修改

**Review by PetrelYy — APPROVED** [review]

**Review by chqy99 — COMMENTED** [review]
最大单点误差和综合单点误差分母不应该有sum

File: docs/MLU-OPS-Accuracy-Acceptance-Standard.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ $count_1$ 表示 $evaluated\\_data$ 大于 $baseline\\_data$ 的个数， $count
 
 ### 2. 算子精度分类
 
-不同类型算子的误差度量方式和精度验收标准不同，我们把 MLU-OPS 算子根据使用的指令、计算的逻辑做了算子类型划分 :
```
Comment (chqy99) [inline docs/MLU-OPS-Accuracy-Acceptance-Standard.md:92]: 最大单点相对误差和综合单点误差的分母不应该出现求和

Comment (Unireverse) [reply docs/MLU-OPS-Accuracy-Acceptance-Standard.md:92]: 改了

**Pull request merged**